### DISCUSSION TITLE
Add Dai as wallet currency (add conversion in wallet to cryptocurrencies)

### DISCUSSION THREAD

**Discussion by leafcutterant** [status-im/status-legacy#12628] [Ideas]
# Feature issue

### User Story

As a user, I want to have Dai as one of the **wallet currencies** ( = a currency you measure value in) so that, in the spirit of decentralization, people can use the wallet without seeing any fiat notions. This also helps cementing Dai as a world currency and getting people used to measuring value in Dai, which will be important when Dai will move away from the USD peg to something better.

### Acceptance Criteria

* displaying in Dai working just as in any other wallet currency
* Dai here meaning Multi-Collateral Dai (MCD)
* DAIUSD value actually pulled from an on-chain or off-chain source, not just a replacing of the USD symbol

**Comment by acolytec3** [comment]
@leafcutterant This will be supported in v1.   [This](https://github.com/status-im/status-react/pull/9496) PR added support and it's in the nightly builds already, just not in the beta app in the app stores.

**Comment by leafcutterant** [comment]
@acolytec3 I'm glad you're updating the *token registry*, but this issue is about something else.

You can select the value of your wallet to be displayed in fiat currencies (USD, INR, etc.). What I want is to have Dai among these currencies. So when I have 1 ETH and 100 Dai, it will read something like "225 DAI" (not exactly, because both  ETHUSD *and* DAIUSD can change). And the value of each stored volatile asset, and transaction fee equivalents, and anything for which the wallet currency is used, would be displayed in Dai as well.

**Tl;dr**: let's be able to have no dirty stinkin fiat on the screen and get us used to accounting in Dai.
